src/encoding_implementations/distance_pair_encoding.jl

```diff
@@ -0,0 +1,74 @@
+export PairDistanceEncoding
+
+using Distances: Euclidean
```

Comment:
```suggestion

using Distances: Chebyshev
```
we use this distance here.